src/i18n/config.ts

```diff
@@ -0,0 +1,29 @@
+import i18next from 'i18next';
```

Comment:
I thought you said we didn't need a backend anymore and that it would embed the translations into the JS?

Reply:
@aburd Sorry, my explanation was probably not clear.
If you refer to https://github.com/i18next/i18next-http-backend.
What it does is run a small server along your application to consume the translation files.
This does not require any special configurations on your dev environment or production (I have used in several projects in the past).

This can also be replaced with a CDN package if you prefer to not have it running in the code.
This is just to simplify translations load, however if you do not want them to run through a server, it can be setup differently.

Reply:
Thanks for the explanation! I got it. I'll resolve this.